endpoints/auction_test.go

```diff
@@ -349,7 +349,7 @@ func TestCacheVideoOnly(t *testing.T) {
 		t.Fatal(err.Error())
 	}
 	syncers := usersyncers.NewSyncerMap(cfg)
-	gdprPerms := gdpr.NewPermissions(nil, config.GDPR{
+	gdprPerms := gdpr.NewPermissions(context.TODO(), config.GDPR{
```

Comment:
Perhaps add a comment to say that `context.TODO()` is being used to generate a dummy context rather than as a placeholder for an improved permissions generator or an improvement of the testing code.

Reply:
Actually should just use `context.Background()`, unless there is some subtle detail that makes background different from TODO, and not work here.